accord-core/src/main/java/accord/local/cfk/PostProcess.java

```diff
@@ -122,6 +128,31 @@ void doNotify(SafeCommandStore safeStore, Key key, NotifySink notifySink)
         }
     }
 
+    static CommandsForKeyUpdate notifyPreBootstrap(CommandsForKeyUpdate update)
```

Comment:
OK you added a comment which is that pre-bootstrap can execute, but what does this mean from the perspective of the integration? It will be asked to attempt to execute transactions that are pre-bootstrap against a state that is ???